language/move-prover/src/prelude.bpl

```diff
@@ -519,6 +519,27 @@ procedure {:inline 1} BorrowField(src: Reference, f: FieldName) returns (dst: Re
     dst := Reference(l#Reference(src), p);
 }
 
+procedure {:inline 1} GetGlobal(address: Value, ta: TypeValue) returns (dst: Value)
```

Comment:
Per convention, should be $GetGlobal here and elsewhere.

Reply:
I have added $ before the names of all Boogie procedure that are used in the translation.